src/snowflake/snowpark/functions.py

```diff
@@ -3636,3 +3650,49 @@ def sproc(
             execute_as=execute_as,
             strict=strict,
         )
+
+
+# Add these alias for user code migration
+call_builtin = call_function
+builtin = function
+countDistinct = count_distinct
+substr = substring
+to_varchar = to_char
+expr = sql_expr
+date_format = to_date
+first = any_value
+create_map = object_construct
+map_keys = object_keys
+monotonically_increasing_id = seq8
+from_unixtime = to_timestamp
+
+
+def asc(e: ColumnOrName) -> Column:
+    """Returns a Column expression with values sorted in ascending order."""
+    c = _to_col_if_str(e, "asc")
+    return c.asc()
+
+
+def desc(e: ColumnOrName) -> Column:
+    """Returns a Column expression with values sorted in descending order."""
+    c = _to_col_if_str(e, "desc")
+    return c.desc()
+
+
+def unix_timestamp(e: ColumnOrName, fmt: Optional["Column"] = None) -> Column:
```

Comment:
I suggest we discuss with more people whether we add these function aliases at the client while Snowflake SQL doesn't have them. An option is to add them to both the client and SQL.